tests/test_aws.py

```diff
@@ -954,9 +963,24 @@ def test_info(self):
             "audience": AUDIENCE,
             "subject_token_type": SUBJECT_TOKEN_TYPE,
             "token_url": TOKEN_URL,
+            "token_info_url": TOKEN_INFO_URL,
             "credential_source": self.CREDENTIAL_SOURCE,
         }
 
+    def test_token_info_url(self):
```

Comment:
add a test for when custom token info url is provided?

Reply:
Similarly in other credential tests as well

Reply:
Done.  Also added a test with an invalid token info url